### DISCUSSION TITLE
I'm a verified student but Copilot isn't free for me

### DISCUSSION THREAD

**Discussion by magliocchetti** [community/community#19084] [Copilot Conversations] [answered]
Hello, I'm a university student which is verified as student on GitHub and it shows everything is good. I even tried to reverify myself [here](https://education.github.com/discount_requests/student_application) but it doesn't let me click "Continue" (even if I fill the textbox) because it says it's not needed:

![copilot_proof1](https://user-images.githubusercontent.com/62136803/174888882-adb9a850-be18-4138-82d1-4ac5d9ed71fa.jpg)

I'm even GitHub Pro thanks to student benefits and I was even successfully [merged](https://github.com/education/GitHubGraduation-2022/pull/8113) for GitHub Graduation 2022. Maybe it's because I'm in GitHub Graduation 2022 and you think that I'm no longer a student? I'm graduating in a bachelor's degree but I'm still a student and I'll be for at least two more years because of a master's degree.

I'm a student and I'm ready to submit any academic document needed.

I'll wait for someone from GitHub staff to help me... I just hope this is a temporary error, I don't want to lose access to Copilot. It really helps me, especially for university projects.

EDIT: looking at posts in this community, this looks like a widespread issue.

**Comment by masonbesmer** [comment]
Looks like this might be an issue on github's end. I have this issue too.
`[INFO] [auth] [2022-06-21T20:18:47.192Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-06-21T20:18:47.194Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
`

**Reply by InukaAsith** [reply]
i also have the same issue

**Reply by InvalidSE** [reply]
I'm having the same issue now, did anyone find a fix?

**Reply by CoffeeLink** [reply]
i also have the same issue.

**Reply by labihi** [reply]
I have the same issue aswell, luckily it's not only me

**Reply by HarshitSohaney** [reply]
Same issue

**Reply by rohanrama** [reply]
same issue

**Comment by Phoenix6296** [comment]
I am also facing a similar problem.
`[INFO] [auth] [2022-06-21T20:56:05.009Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-06-21T20:56:05.018Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"`

**Comment by ldeluigi** [comment]
Duplicate of #19050?

**Reply by masonbesmer** [reply]
yup

**Comment by magliocchetti** [comment, marked as answer]
Retried after some hours and it's apparently fixed. Thank you!

![Screenshot 2022-06-22 020303.jpg](https://github.zendesk.com/attachments/token/6DQCob2tw1cQOwT0LpoFeprFC/?name=Screenshot+2022-06-22+020303.jpg)

As noted by another user, this thread is a dup of #19050 . So if you still have this issue I think it's better to go to that thread to keep things more organized (?). Sorry for that!

**Reply by RaziehSh1987** [reply]
hello,i have the same problem, where is thread?

**Reply by magliocchetti** [reply]
https://github.com/orgs/github-community/discussions/19050

**Reply by dayglojago** [reply]
This link is broken.

**Reply by GenericallyNamed** [reply]
The forum link is also broken for me. I visited the same url as you, github.com/github-copilot/free_signup, and it redirects me to a page with payment options. And yet, if I visit education.github.com/benefits, it says clear as day that I am a verified student. I'm going to try verification again since in my case I was verified as a highschool student, but now I am in university, so perhaps re-verification using a different method will be my solution.

**Comment by Folashy** [comment]
Hello, 
![Screenshot 2022-08-22 at 14 31 54](https://user-images.githubusercontent.com/100294216/186096275-f5a82c20-6f02-4bb3-95a7-b7db40a4de72.png)
I'm a student of Decagon Institute and could not use the free copilot student developer pack. It is requesting for payment

I'll wait for someone from GitHub staff to help me... I need the access to Copilot. It has been really helpful to my co students especially for our projects.

**Comment by GenericallyNamed** [comment]
In my case, I was initially verified already as a highschool student, and when I visited education.github.com/benefits?type=student, the page said that I was a verified student. Now that I am a university student, I tried re-verifying using my university ID, after which I was able to access GitHub Copilot. 
It's hard to say this will work for everyone, but it might be worth trying to re-verify. However, in my case it's hard to say it was simply from re-verifying or if switching to a different verification fixed it, since GitHub itself was already saying that I was verified as a student (but wouldn't grant me access to student benefits).

**Comment by nikrozman21** [comment]
This issue seems to have resurfaced for me too.

**Reply by atahanoezer** [reply]
I am also having the same issue

**Reply by realhen** [reply]
Me too

**Reply by patrickjcraig** [reply]
Oh no did they remove it from the student package

**Reply by AdamBajger** [reply]
I hope they did not remove it entirely...

**Reply by Ramy-Ashraf** [reply]
I have the same issue

**Reply by realhen** [reply]
Nvm i was randomly reverified

**Reply by juac08** [reply]
I also have same issue

**Reply by Ramy-Ashraf** [reply]
![image](https://user-images.githubusercontent.com/85342902/206880042-151655c9-1ded-41d9-94d9-9d8e5dfff657.png)

seems it is still included

**Reply by juac08** [reply]
You were last verified as a student on Aug 10, 2022. It is not necessary for you to re verify at this time. But still unable to use.

**Reply by benthecoder** [reply]
Having the same issue

**Reply by thanhtradev** [reply]
I have the same issue recently

**Reply by mfaseehuddin** [reply]
having the same issue

**Reply by juac08** [reply]
it starts working

**Reply by patrickjcraig** [reply]
Works for me now

**Comment by juac08** [comment]
Seems like working. Just reinstall the extension

**Reply by thanhtradev** [reply]
Yes, this works correctly after a moment LOL

**Reply by AdamBajger** [reply]
Everyone just panicked so hard :D The world almost burned down :D

**Comment by Akash1134** [comment]
👋 Hi there, Thank you everyone for taking the time to report this issue.

Apologies for that temporary issue on our end. Please retry logging in to your IDE plugin, and Copilot should work as expected. 

If you run into any further issues, please reply back in detail with any useful screenshots.

**Reply by patrickjcraig** [reply]
Are there any plans for Copilot to be removed from the student developer package in the future?

**Comment by AmeliaTYR** [comment]
Hello, in my case I got the Technical Preview which expired, but I am still a student for another year. I tried to reverify my school email and reinstall the plugin but got the following message: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended" Is it possible for me to get the copilot student access?

**Reply by esoterik-dev** [reply]
Facing the same issue. I still have my Github Student Developer Pack active but Copilot still asks me for a subscription. This is what the extension reports:
```
[INFO] [auth] [2023-01-08T19:00:32.319Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-01-08T19:00:32.320Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```

**Comment by HikaruChang** [comment]
I had the same problem. It stopped working just now. But my student certification has not expired.

![DB675F5D-ACB5-4F43-8653-BC6D83623935](https://user-images.githubusercontent.com/16264281/211208919-400685e8-bd11-4767-be2a-d1567c9520aa.png)
![E4977FB2-8967-4A7F-BCDC-017B1C9F4C4C](https://user-images.githubusercontent.com/16264281/211208914-d760cc82-7b18-4175-966c-ec3f4d389373.png)

**Reply by HikaruChang** [reply]
I received an email, which has only been renewed for one month

![1B6A4AEF-55B5-4945-8B76-C3696F2A3EF0](https://user-images.githubusercontent.com/16264281/211234121-22c27fd8-81e8-487e-a44f-ccd2c14e3491.jpeg)
![3751589B-B204-44C4-8EC3-C43AA71D67E3](https://user-images.githubusercontent.com/16264281/211234124-002708e1-89ee-4a83-8eee-ee6c51617041.jpeg)

**Reply by HikaruChang** [reply]
I seem to have found the answer.
![17D26F6C-8531-4D4E-B689-3B48B323FF04](https://user-images.githubusercontent.com/16264281/211235513-57d393d7-064c-4986-b48c-c9bed1e2629f.jpeg)

**Comment by OmKshirsagar** [comment]
its happened again

**Comment by otkrickey** [comment]
I have created a discussion for today's case. Please share your situation. #43634

**Comment by oceceli** [comment]
Yeah here we go again :)

**Comment by zoranduric** [comment]
I have same problem even I'm verified student.

**Reply by otkrickey** [reply]
I have created a discussion for today's case. Please share your situation. #43634

**Comment by waleedrana777** [comment]
Ahh shit, here we go again!

**Comment by ataberkw** [comment]
Lol here we go again. Now it feels like I can't code without copilot. Don't leave us alone after we get addicted.

**Comment by iamsabbirsobhani** [comment]
I am getting the following error, while accessing copilot, which was good before 10 hours. I am verified `Student Developer Pack` holder of GitHub.
```
[INFO] [auth] [2023-01-09T05:33:32.615Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-01-09T05:33:32.621Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found."
```
While trying to enable `copilot` from the GitHub billing page, I am being shown sign up page with price.

**Comment by otkrickey** [comment]
Check #43634

**Comment by razvyrazvy41** [comment]
i have the same log as everyone 
[INFO] [auth] [2023-01-09T07:24:42.934Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-01-09T07:24:42.937Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
I am also as everyone a verified student.
Please help!

**Comment by kevinlu1248** [comment]
Broken for me too.

**Comment by Siyx1** [comment]
I have the same problem.

**Comment by duheidm354** [comment]
I have the same problem.

**Comment by zslzx** [comment]
I have the same problem.

**Comment by wukailu** [comment]
same problem

**Comment by harshpathakzz** [comment]
I am unable to start GitHub copilot when I click on start free trial, it redirects to the page.
I have GitHub student developer pack as well
![image](https://user-images.githubusercontent.com/90128982/230757247-45eb7fc2-df84-478e-99b3-04db36bef556.png)

**Comment by LucasNavarroB** [comment]
Hello, I wanted to start using Copilot and when I log in, it sends me the payment page, and as I understand it, it is free for verified students.
![imagen](https://user-images.githubusercontent.com/49734933/234967426-903d6112-2fae-46b9-8cce-5fb0bd0e4701.png)

**Comment by FloXPD** [comment]
This link worked for me: github.com/github-copilot/free_signup
Make sure you go to free-signup and not to signup, there you can just request it again. At least for me, if it doesn't work right away try a little later.

**Comment by InstantlyMoist** [comment]
Still occurs for me
